userland/examples/radio_rx/main.c

```diff
@@ -0,0 +1,33 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include <led.h>
+#include <radio.h>
+#include <timer.h>
+
+#define BUF_SIZE 60
+char packet_rx[BUF_SIZE];
+char packet_tx[BUF_SIZE];
+bool toggle = true;
+
+static void callback(__attribute__ ((unused)) int unused0,
```

Comment:
We talked about this offline, but to capture notes from the call:

  - `__attribute__((unused))` is clear and semantically meaningful, `(void)` is not
  - The unused attribute is supported by both gcc and llvm, which are our only realistic compiler targets in the anywhere near future
  - A compromise we could find/replace for in the future would be an `UNUSED` macro that expands. I would probably still expand it to the attribute rather than the `(void)` [with the note that since we're in a macro anyway, compiler checks are an option for non-gcc/llvm to expand to the void form], on the off chance that gcc/llvm can do (or may someday do) something more intelligent with the knowledge gained from the unused attribute over just the `(void)` convention
     - thus, for now: just keep going with the attributes

I would actually say the `UNUSED` macro is the most common thing I've seen. I rarely see code with the `(void)`, but obviously ymmv